src/applications/check-in/tests/e2e/commands/index.js

```diff
@@ -57,38 +57,50 @@ Cypress.Commands.add('createScreenshots', filename => {
     cy.wait(1000);
     // Back to english
     cy.get('[data-testid="translate-button-en"]').click();
-    const screenshotHiddenItem = (item, type) => {
+    const expandHiddenItem = item => {
       item.each((i, v) => {
         cy.get(v)
           .shadow()
           .find('[aria-expanded="false"]')
           .click();
       });
-      cy.screenshot(`english/${filename}-${type}-expanded`);
+      cy.wait(1000);
+    };
+    // Expand additional info if it exists and make more screenshots
+    // eslint-disable-next-line cypress/no-assigning-return-values
```

Comment:
Instead of having three `eslint-disable-next-line` calls in a row, you could `eslint-disable` and then after your assignments do `eslint-enable`. Just a thought, may be a personal preference to have one ESLint bot error instead of three. ;)

Reply:
oh that is a good idea! hit the merge button too quick on this one, but I'll add it to some future tech debt. thanks!